bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties

```diff
@@ -17,8 +17,11 @@ channel-type.system.location.description = Ort in geographischen Koordinaten (Br
 channel-type.system.motion.label = Bewegung
 channel-type.system.motion.description = Zeigt eine erkannte Bewegung an.
 channel-type.system.brightness.label = Helligkeit
+channel-type.system.brightness.description = Kontrollieren Sie die Helligkeit und schalten Sie das Licht ein und aus.
```

Comment:
I would rather not address the user, i.e. "sie" directly, but instead describe it like on the other channels: Just what can be done with the channel. So here I would recommend: "Steuert die Helligkeit und schaltet das Licht ein und aus."